scipy-1.0/paper.tex

```diff
@@ -461,15 +461,15 @@ \subsection*{SciPy Begins}
 of lines of C++, Python, and Fortran 95, all working together just as we hoped,
 doing compute-intensive calculations on massively parallel computers.
 \end{quote}
-While it is now more commonly accepted, Paul took the time to explain to
+While it is now more commonly accepted, Paul Dubois took the time to explain to
```

Comment:
Since the previous sentence was about about him, I think that just "Dubois" is fine. But yes, either is better than "Paul".

Reply:
ok, switched to surname only